flyte.config

```diff
@@ -0,0 +1,10 @@
+[sdk]
+# This option specifies which workflow packages should be serialized during in-container serialization.
+workflow_packages=myapp.workflows
+
+[auth]
+# This default service account is added to workflow executions and must at have read and write privileges
```

Comment:
```suggestion
# Uncomment if you want to use a service account for all your tasks and workflow executions. This service account should be created by you and available on the k8s cluster and it will be used to read and write data from the backend store like S3/GCS, or to connect to any services that you use in your tasks.
```